### PR TITLE
update Georgian language pack

### PR DESCRIPTION
**Pull request by Landish**

### CODE REVIEW

File: src/ka/validation.php [hunk 1]
```diff
@@ -92,7 +89,7 @@
     |
     */
 
-    'custom'               => [
+    'custom' => [
```
Comment (caouecs) [inline src/ka/validation.php]: Problem of style

File: src/ka/validation.php [hunk 2]
```diff
@@ -109,8 +106,6 @@
     |
     */
 
-    'attributes'           => [
-        //
-    ],
+    'attributes' => [],
```
Comment (caouecs) [inline src/ka/validation.php]: Don't change it

File: src/ka/validation.php [hunk 3]
```diff
@@ -109,8 +106,6 @@
...
 
-];
+];
```
Comment (caouecs) [inline src/ka/validation.php]: Need new line at end

File: src/ka/validation.php (hunk 2 above)
Comment (Landish) [inline src/ka/validation.php]: Laravel's core validation.php file has been changed and for some reason I changed it as well. https://github.com/laravel/laravel/blob/master/resources/lang/en/validation.php#L112

**Comment by caouecs** [comment]
Thanks

**Pull request merged**